src/chapters/exploratory-data-analysis/eda.rst

```diff
@@ -1,75 +1,77 @@
 =========================
 
-Read the following articles, follow along where instructed:
+Read the following articles, follow along where instructed.  You do not need to install ``pandas``, it comes with ``Anaconda``.  
 
-.. admonition:: tip
+.. admonition:: Tip
   
-  For Medium articles: if you run out of free articles, open the page in an incognito window
+  For Medium articles: if you run out of free articles, open the page in an incognito window.
 
 Exploring Data with Python
 --------------------------
 
-1. `Exploratory data analysis in Python <https://towardsdatascience.com/exploratory-data-analysis-in-python-c9a77dfa39ce>`__.
+Code along with this article.  
+
+Stop at Step #8 "Detecting Outliers".
+
+#. `Exploratory data analysis in Python <https://towardsdatascience.com/exploratory-data-analysis-in-python-c9a77dfa39ce>`_.
+
 
-* Key Takeaways: Outline of common steps used in EDA, note that there is no one process for performing EDA, 
-  it all depends on the dataset and your questions.
-* Just read and follow the steps for comprehension, don't need to do the tutorial.
-* Stop at #8: “Detecting Outliers”
 
 Get to Know Your Data
 ---------------------
 
-2. `Getting to know your data <https://medium.com/@shanegary/getting-to-know-your-data-9e42935e7f60>`__.
+Read the following articles.
+
+#. `Getting to know your data <https://medium.com/@shanegary/getting-to-know-your-data-9e42935e7f60>`_.
+
+#. `Data Types in Statistics <https://towardsdatascience.com/data-types-in-statistics-347e152e8bee>`_.
 
-* Key Takeaways: Questions to better understand why the data was collected.
-* *Suggested Reading:* `Data Types in Statistics <https://towardsdatascience.com/data-types-in-statistics-347e152e8bee>`__.
 
-  * Key Takeaways: Definitions for discrete, continuous and categorical data.
+Python ``pandas``
+-----------------
 
-Python Pandas
--------------
+Code along with this article.  
 
-3. `Python Pandas Tutorial: A Complete Introduction for Beginners <https://www.learndatasci.com/tutorials/python-pandas-tutorial-complete-introduction-for-beginners/>`__.
+Stop at "Handling Duplicates".
 
-* You do not need to install pandas, it comes with the Anaconda package.
-* Try coding along with the article 
-* Stop at “Handling Duplicates” header
+#. `Python Pandas Tutorial: A Complete Introduction for Beginners <https://www.learndatasci.com/tutorials/python-pandas-tutorial-complete-introduction-for-beginners/>`_
 
-Statistics in Pandas
---------------------
+Statistics in ``pandas``
+------------------------
 
-4. `Basic statistics in pandas DataFrame <https://medium.com/@kasiarachuta/basic-statistics-in-pandas-dataframe-594208074f85>`__.
+Read the following article.
 
-* Key takeaway: Using pandas DataFrame; with examples
+#. `Basic statistics in pandas DataFrame <https://medium.com/@kasiarachuta/basic-statistics-in-pandas-dataframe-594208074f85>`__.
   
-What is a Dataframe?
---------------------
+What is a ``DataFrame``?
+------------------------
 
```

Comment:
i went with lowercase because that is how the pandas documentation defines itself.  i prefer to match the documentation myself, and believe that as long as we are consistent in our learning materials then it will be okay.  Open for discussing this as a team.